packages/block-editor/src/store/defaults.js

```diff
@@ -131,6 +132,9 @@ export const SETTINGS_DEFAULTS = {
 		{ slug: 'full', name: __( 'Full Size' ) },
 	],
 
+	// Allow plugin to disable Image Editor if need be
+	disableImageEditor: false,
```

Comment:
What's if it's the opposite: `imageEditing: true`

Reply:
It looks like the other block-editor settings use `disable` for example custom colors, font sizes, and gradients. I followed their lead on that.

Reply:
These things may change once global styles is here. imageEditing allow for potentially having a more advanced config later (object).

Reply:
@youknowriad I'm fine with switching. Is it ok to create it as `imageEditing` and keep it as a boolean, defaulting to `true`?

We would then be changing its type down the line if we add settings, but setting to false would work. If create it as an object, it will be empty to start, which is falsey, so tricky to test for disabled.